table/init.go

```diff
@@ -74,3 +78,14 @@ func Configure() {
 func SetCsCapacity(capacity int) {
 	csCapacity = capacity
 }
+
+func CreateFIBTable(fibTableAlgorithm string) {
+	switch fibTableAlgorithm {
+	case "hashtable":
+		m := core.GetConfigUint16Default("tables.fib.hashtable.m", 5)
+		newFibStrategyTableHashTable(m)
+	default:
```

Comment:
I suggest you put `nametree` here, and let the default branch to `logFatal`.
You have already set the default value when user is using an old configuration file that omits this parameter. If the control flow falls into default branch, it means the user specified a non-existing algorithm. It's better to report error.